windIO/IEA-22-280-RWT_Floater.yaml

```diff
@@ -982,28 +982,28 @@ airfoils:
-              values: [0.0, 0.09142857142857111, 0.18285714285714336, 0.27428571428571447, 0.36571428571428555, 0.3810049135709473, 0.35061277528446283, 0.320220636997978, 0.28982849871149347, 0.2707250005663944, 0.2854874228454521, 0.30024984512451014, 0.31501226740356797, 0.329774689682626, 0.34069766801241747, 0.35950142449830097, 0.37812367027102983, 0.3942955214161714, 0.41046737256131277, 0.4249753742894562, 0.4386076657981268, 0.45098359876857597, 0.45941097633318584, 0.46783835389779577, 0.47031385081975385, 0.47154937260782614, 0.46958450526506207, 0.46308575332027985, 0.45658700137549757, 0.43799458507675076, 0.419402168778004, 0.39785952371705374, 0.3742343642357246, 0.3500927041748871, 0.32347184133240964, 0.29685097848993225, 0.26985588137702365, 0.2427417097235234, 0.2161238741782509, 0.19044907723861126, 0.16477428029897165, 0.10903179415080097, 0.05106207176496147, 0.05325678844586054, 0.0017455604385143639, -0.017518182266531845, -0.009771024583074717, -0.01280632568675133, -0.01379208857165154, -0.0234359375783005, -0.043419123987402275, -0.057449318143686756, -0.06777949527749177, -0.07876899508338389, -0.0845911882791724, -0.08706052660304915, -0.08937313897352425, -0.09162886966564893, -0.09379772996377075, -0.09591368318861304, -0.09791309471362838, -0.09974476631894746, -0.101158427779519, -0.10218744765001814, -0.1026058686918423, -0.10274366627370718, -0.10236370310093568, -0.10172049092360347, -0.10080851951577674, -0.0989474361500563, -0.09634808326238653, -0.09358514581721361, -0.0905544804454691, -0.08754514225338139, -0.09216213670758056, -0.11222254975971548, -0.13213418905476543, -0.12815178898156315, -0.1561356603502806, -0.184038244942224, -0.21084345804771643, -0.23764867115320884, -0.26357232433849104, -0.28903199861839934, -0.31401063920721095, -0.3374774596240042, -0.36094428004079726, -0.3820594439364129, -0.40268467939011643, -0.4219430489508378, -0.4392650251631677, -0.45658700137549757, -0.46308575332027985, -0.4695845052650621, -0.4715493726078262, -0.4703138508197539, -0.46783835389779577, -0.4594109763331859, -0.45098359876857597, -0.43860766579812693, -0.4249753742894562, -0.41046737256131277, -0.3942955214161714, -0.37812367027102983, -0.35950142449830097, -0.3406976680124175, -0.329774689682626, -0.31501226740356797, -0.30024984512451014, -0.2854874228454521, -0.27072500056639437, -0.30697135585435, -0.3602206369979779, -0.41346991814160516, -0.4667191992852331, -0.45714285714285696, -0.3428571428571431, -0.22857142857142915, -0.11428571428571388, 0.0]
+        #  -  configuration: Fully turbulent flow
```

Comment:
these polars should not be commented out.

Reply:
Yep, this was not necessarily meant to be pushed.  Our software doesn't like that there's multiple polars with the same `Re`.  @ptrbortolotti, how do you think we should handle this long term?

Reply:
I don't like the multiple sets of polars because we don't say anywhere which of the three we are using. for the time being, I'm commenting out the extra two sets locally, whereas @gbarter just pushed a quick workaround to WISDEM where if multiple sets a defined, the first one is picked. This to me is a risky assumption and I would get rid of the extra two sets, which could be saved in a standalone yaml file that users interested in fully clean or fully rough polars could be pointed to. @fzahle what do you think? @dzalkind, still, please uncomment those lines in this PR

Reply:
Will do. That's good to know. 

I'm also switching this to a draft PR while I work out the last kinks in the mooring design.  It's not quite ready yet but hopefully later today.

Reply:
@ptrbortolotti this seems to be more a WindIO discussion. If you don't think adding polars with different surface conditions is the intended use of the `airfoils` section in the schema, and it should rather be reserved for only one dimension, Reynolds number, then that should be specified. In our tools we default to the first polar but the user can for each defined airfoil pick or interpolate in the available polars according to user-defined dimensions. If you for WISDEM think it's risky to default to the first one, you can force users to define which polars to use if multiple polars are available.